packages/eslint-config-react-app/index.js

```diff
@@ -155,6 +155,15 @@ module.exports = {
     'unicode-bom': ['warn', 'never'],
     'use-isnan': 'warn',
     'valid-typeof': 'warn',
+    'no-restricted-properties': ['warn', {
+      object: 'require',
+      property: 'ensure',
+      message: 'Please use import() instead.'
```

Comment:
People seeing this warning might not even be aware `import()` exists. We should link to some section that clearly explains how to use it.